rules/windows/registry_event/sysmon_runonce_persistence.yml

```diff
@@ -0,0 +1,25 @@
+title: Run Once Task Configuration in Registry
+id: c74d7efc-8826-45d9-b8bb-f04fac9e4eff
+description: Rule to detect the configuration of Run Once registry key. Configured payload can be run by runonce.exe /AlternateShellStartup
+author: 'Avneet Singh @v3t0_, oscd.community'
+status: experimental
+date: 2020/10/18
+references:
+    - https://twitter.com/pabraeken/status/990717080805789697
+    - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OSBinaries/Runonce.yml
+tags:
+    - attack.defense_evasion
+    - attack.t1112
+logsource:
+    product: windows
+    category: registry_event
+detection:
+    selection:
+      EventID: 13
```

Comment:
Hello @V3T0! Are you sure that you need eventID here? Shouldn't it be already covered by the `category: registry_event`?

Reply:
I assume that `category: registry_event` would have both sysmon EventID 12 and 13, that's why I included Event ID in this.

But we can skip the EventID here since I've mentioned EventType as "Set Value" in the rule, which is specific to Event ID 13. 
Do you think I should remove EventID from here @yugoslavskiy or should we keep it as there is no impact on the logic? 

It might have some impact on SIEM side (while evaluating more fields), I am not sure about that tbh. :)

Reply:
If you will try to convert this rule to an actual query, you will get the answer (: 
It is done by the converter, so the eventID is redundant. Try it yourself and you will see

Reply:
Hello @V3T0! Could you please tell me if you are going to proceed with the rule development? (:

Reply:
Hi @yugoslavskiy, yes I'm going to look into the rule again and your suggestion about the rule conversion. I'll get back to you soon :)

Reply:
Hello @V3T0! (: Is there any update on the rule development?

Reply:
Thank you!